core/src/integration-test/java/com/scalar/db/api/DistributedStorageConditionalMutationIntegrationTestBase.java

```diff
@@ -70,15 +70,15 @@ public abstract class DistributedStorageConditionalMutationIntegrationTestBase {
           .build();
 
   private static final int ATTEMPT_COUNT = 5;
-  private static final Random random = new Random();
-
   private static final int THREAD_NUM = 10;
 
   private DistributedStorageAdmin admin;
   private DistributedStorage storage;
   private String namespace;
 
   private long seed;
+  private ThreadLocal<Random> random;
```

Comment:
We should use `ThreadLocal` for `Random` because random can be accessed by multi threads.